dev/cli

```diff
@@ -46,6 +47,24 @@ function enable_oidc() {
   env_args="$env_args $keycloak_env_args"
 }
 
+function enable_azure() {
+
+  . ../ci/authn-azure/check_dependencies.sh
+
+  echo "Setting Azure details as env variables"
+
+  local azure_env_args=''
+  azure_env_args="$azure_env_args -e AZURE_TENANT_ID=$AZURE_TENANT_ID"
+  azure_env_args="$azure_env_args -e AZURE_AUTHN_INSTANCE_IP=$AZURE_AUTHN_INSTANCE_IP"
+  azure_env_args="$azure_env_args -e AZURE_AUTHN_INSTANCE_USERNAME=$AZURE_AUTHN_INSTANCE_USERNAME"
+  azure_env_args="$azure_env_args -e AZURE_AUTHN_INSTANCE_PASSWORD=$AZURE_AUTHN_INSTANCE_PASSWORD"
+  azure_env_args="$azure_env_args -e AZURE_SUBSCRIPTION_ID=$AZURE_SUBSCRIPTION_ID"
+  azure_env_args="$azure_env_args -e AZURE_RESOURCE_GROUP=$AZURE_RESOURCE_GROUP"
+  azure_env_args="$azure_env_args -e USER_ASSIGNED_IDENTITY=$USER_ASSIGNED_IDENTITY"
+#  azure_env_args="$azure_env_args -e SYSTEM_ASSIGNED_IDENTITY=$SYSTEM_ASSIGNED_IDENTITY" # TODO: add this once available
+  env_args="$env_args $azure_env_args"
+}
+
 unset COMPOSE_PROJECT_NAME
```

Comment:
curious what this does?

Reply:
not sure tbh

Reply:
Google it and tell me? :)

Reply:
`COMPOSE_PROJECT_NAME` is the prefix used to name containers when created via docker-compose

Reply:
Oh right, it sat somewhere in the back of my head. thanks!